packages/stylist-macros/src/output/cow_str.rs

```diff
@@ -0,0 +1,43 @@
+use super::{ContextRecorder, Reify};
+use proc_macro2::{Span, TokenStream};
+use quote::{quote, quote_spanned, ToTokens};
+use syn::{spanned::Spanned, LitStr};
+
+#[derive(Debug)]
+pub enum OutputCowString {
```

Comment:
What's the advantage of stripping this out of `OutputFragment`?

Whilst the code size of procedural macros are not as critical as other parts, I still wish to keep the code footprint low by not having unnecessary abstractions.

Reply:
`OutputFragment` returns an expression of type `StringFragment`, `OutputCowString` returns one of type `Cow<'static, str>` from `into_token_stream`. See above why `OutputCowString` is also used in `OutputAttribute`.